src/components/Board.vue

```diff
@@ -137,6 +156,39 @@ export default class Board extends Vue {
   addArrow (): void {
     this.$store.dispatch(`${StoreModuleEnum.arrowStore}/${EActionArrow.ADD_DATA}`)
   }
+
+  @Watch('storeArrows')
+  async watchArrowsExists (newValue: IArrowData[]) {
+    if (newValue.length === 0 && !this.isActive) {
+      try {
+        const gameData = this.getGameData()
+        this.resetScore()
+        await this.$store.dispatch(
+          `${StoreModuleEnum.userStore}/${EActionUser.UPDATE_DB_SCORE}`, gameData
+        )
+      } catch (e) {
+        throw new Error(e.message)
+      }
+    }
+  }
+
+  getGameData (): IFirestoreUserScore {
```

Comment:
Could you keep this data in the store? Looks like this component doesn't really need it, it gets data from the store, parses it, and returns to the store. 
Please, correct me, if I understand it wrong

Reply:
@silchencko thanks, 
You are rigth, I transfered this fuunction to the store in [this commit](https://github.com/what-the-hub/vue-game/pull/37/commits/bb42cb19acff96dcf96195aa3545e743f4f2d779)

Reply:
Maybe `getGameData` can be removed?

Reply:
@silchencko oh thanks. Yes, of course. This [commit ](https://github.com/what-the-hub/vue-game/pull/37/commits/444ec76dac48c9a9f42a2774aa00084774eb63ab)will fixed it